docs/quickstart.md

```diff
@@ -62,6 +62,51 @@ sudo DEBIAN_FRONTEND=noninteractive apt-get \
      docker-ce aufs-tools-
 sudo usermod -aG docker $(whoami)
 
+# Install device-mapper
+sudo DEBIAN_FRONTEND=noninteractive apt-get install -y dmsetup
+```
+
+A similar script to install dependencies for rpm based linux distro e.g. Amazon Linux 2 can be found here
+
+<details>
+
+```bash
+#!/bin/bash
+
+cd ~
+
```

Comment:
Do you need `cd` here and below? All commands on the section won't read/write files in the current working directory.